safe/reportv4/test/test_impact_report.py

```diff
@@ -0,0 +1,304 @@
+                    'output_format': 'file',
+                    'output_path': 'analysis-result-output.html',
+                    'template': 'standard-template/'
+                                'jinja2/'
+                                'analysis-result.html',
+                },
+                {
+                    'key': 'analysis-breakdown',
+                    'type': 'Jinja2',
+                    'processor': jinja2_renderer,
+                    'extractor': analysis_detail_extractor,
+                    'output_format': 'file',
+                    'output_path': 'analysis-detail-output.html',
+                    'template': 'standard-template/'
+                                'jinja2/'
+                                'analysis-detail.html',
+                },
+                {
+                    'key': 'action-checklist',
+                    'type': 'Jinja2',
+                    'processor': jinja2_renderer,
+                    'extractor': action_checklist_extractor,
+                    'output_format': 'file',
+                    'output_path': 'action-checklist-output.html',
+                    'template': 'standard-template/'
+                                'jinja2/'
+                                'bullet-list-section.html',
+                },
+                {
+                    'key': 'notes-assumptions',
+                    'type': 'Jinja2',
+                    'processor': jinja2_renderer,
+                    'extractor': notes_assumptions_extractor,
+                    'output_format': 'file',
+                    'output_path': 'notes-assumptions-output.html',
+                    'template': 'standard-template/'
+                                'jinja2/'
+                                'bullet-list-section.html',
+                },
+                {
+                    'key': 'minimum-needs',
+                    'type': 'Jinja2',
+                    'processor': jinja2_renderer,
+                    'extractor': minimum_needs_extractor,
+                    'output_format': 'file',
+                    'output_path': 'minimum-needs-output.html',
+                    'template': 'standard-template/'
+                                'jinja2/'
+                                'minimum-needs.html',
+                },
+                {
+                    'key': 'impact-report',
+                    'type': 'Jinja2',
+                    'processor': jinja2_renderer,
+                    'extractor': impact_table_extractor,
+                    'output_format': 'file',
+                    'output_path': 'impact-report-output.html',
+                    'template': 'standard-template/'
+                                'jinja2/'
+                                'impact-report-layout.html',
+                }
+            ]
+        }
+
+        exposure_json = self.fixtures_dir('analysis_sample/1-exposure.geojson')
+        exposure_layer = QgsVectorLayer(exposure_json, '', 'ogr')
```

Comment:
With your path, you should call this function : https://github.com/inasafe/inasafe/blob/develop/safe/test/utilities.py#L308
It will do the monkey patching for you.